libkbfs/kbpki_client.go

```diff
@@ -271,6 +271,18 @@ func (k *KBPKIClient) IsTeamReader(
 	return teamInfo.Writers[uid] || teamInfo.Readers[uid], nil
 }
 
+// GetTeamRootID implements the KBPKI interface for KBPKIClient.
+func (k *KBPKIClient) GetTeamRootID(ctx context.Context, tid keybase1.TeamID) (
+	keybase1.TeamID, error) {
+	teamInfo, err := k.serviceOwner.KeybaseService().LoadTeamPlusKeys(
+		ctx, tid, UnspecifiedKeyGen, keybase1.UserVersion{},
+		keybase1.TeamRole_NONE)
+	if err != nil {
+		return keybase1.TeamID(""), err
+	}
+	return teamInfo.RootID, nil
```

Comment:
`LoadTeamPlusKeys` doesn't populate `RootID` for sub-teams, so this would be empty for root teams. It seems contradictory with the interface comment (https://github.com/keybase/kbfs/pull/1099/files#diff-9bdcfee86ede222f52d83f6b7f9f7f28R547) which suggests the method can be used for both root teams and sub-teams?